modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -762,8 +762,14 @@ void ONNXCompiled::Run(const std::vector<cv::Mat>& ins,
         // Hard path - run session & user-defined post-processing
         // NOTE: use another list of output names here
         std::vector<const char*> out_names;
-        for (auto &&ti : out_tensor_info) {
-            out_names.push_back(ti.name.c_str());
+        if (params.names_to_remap.empty()) {
+            for (auto &&ti : out_tensor_info) {
```

Comment:
Think about how to remove code duplication, btw you don't need to use loop in there.

Reply:
`ade::util::copy` ?

Reply:
Added ade::util::transform. +-.
I wanted to use one std::transform, but names_to_remap and out_tensor_info can have different sizes.